modules/3d/include/opencv2/3d.hpp

```diff
@@ -1964,7 +1965,7 @@ correctly only when there are more than 50% of inliers.
 CV_EXPORTS_W cv::Mat estimateAffine2D(InputArray from, InputArray to, OutputArray inliers = noArray(),
                                   int method = RANSAC, double ransacReprojThreshold = 3,
                                   size_t maxIters = 2000, double confidence = 0.99,
-                                  size_t refineIters = 10);
+                                  size_t refineIters = 10, double outlierRatio=0.45);
```

Comment:
As documentation states:

```
The following methods are possible:
-   @ref RANSAC - RANSAC-based robust method
-   @ref LMEDS - Least-Median robust method
```

and we already have `ransacReprojThreshold - ... Applies only to RANSAC` parameter.
Adding one more parameter `"Applies only to LMeDS"` with ignoring of another one looks really weird.

It makes sense to split this function on two with clear names:
```
estimateAffine2D_RANSAC(... without "method" and with "ransacReprojThreshold")
estimateAffine2D_LMEDS(... without "method"/"ransacReprojThreshold" and with "outlierRatio")
```

P.S. Creating of "GOD" functions should be considered as a bad practice with a lot of troubles.